packages/fxa-auth-server/test/local/payments/stripe.js

```diff
@@ -249,6 +253,175 @@ describe('StripeHelper', () => {
     });
   });
 
+  describe('createPlainCustomer', () => {
+    it('creates a customer using stripe api', async () => {
+      const expected = deepCopy(newCustomerPM);
+      sandbox.stub(stripeHelper.stripe.customers, 'create').returns(expected);
+
+      const actual = await stripeHelper.createPlainCustomer(
+        'uid',
+        'joe@example.com',
+        'Joe Cool',
+        uuidv4()
+      );
+
+      assert.deepEqual(actual, expected);
+    });
+
+    it('surfaces stripe errors', async () => {
+      const apiError = new stripeError.StripeAPIError();
+      sandbox.stub(stripeHelper.stripe.customers, 'create').rejects(apiError);
+
+      return stripeHelper
+        .createPlainCustomer('uid', 'joe@example.com', 'Joe Cool', uuidv4())
+        .then(
+          () => Promise.reject(new Error('Method expected to reject')),
+          (err) => {
+            assert.equal(err, apiError);
+          }
+        );
+    });
+  });
+
+  describe('retryInvoiceWithPaymentId', () => {
+    it('retries with an invoice successfully', async () => {
+      const attachExpected = deepCopy(paymentMethodAttach);
+      const customerExpected = deepCopy(newCustomerPM);
+      const invoiceRetryExpected = deepCopy(invoiceRetry);
+      sandbox
+        .stub(stripeHelper.stripe.paymentMethods, 'attach')
+        .returns(attachExpected);
+      sandbox
+        .stub(stripeHelper.stripe.customers, 'update')
+        .returns(customerExpected);
+      sandbox
+        .stub(stripeHelper.stripe.invoices, 'retrieve')
+        .returns(invoiceRetryExpected);
```

Comment:
(My brain doesn't know it's a holiday and randomly remember this thing it saw...)

We should change these `stub`s where it `returns` to `resolves` when the stub'd function is an async function.  That way they'll catch errors if the callers did not `await`.